pkg/services/ngalert/remote/alertmanager_test.go

```diff
@@ -137,20 +137,24 @@ func TestApplyConfig(t *testing.T) {
 	})
 
 	var configSent client.UserGrafanaConfig
-	var lastConfigSync, lastStateSync time.Time
+	var configSyncs, stateSyncs int
```

Comment:
It was simpler to count the number of config/statesyncs in the test, so I changed these to ints.